api-authentication/server/index.js

```diff
@@ -59,6 +59,9 @@ app.post('/api/auth/sign-in', (req, res, next) => {
 
       /* your code starts here */
       const [user] = result.rows;
+      if (!user) {
+        throw new ClientError(401, 'user not found');
+      }
```

Comment:
We don't want to give the client a hint that the username is wrong. instead we just say 'Invalid login"

Reply:
![Screen Shot 2022-09-07 at 9 54 38 AM](https://user-images.githubusercontent.com/106217355/188935929-4bac0720-eaf8-4a05-bd3b-e1ebcbd518c4.png)